RayLin.md

```diff
@@ -1,12 +1,32 @@
-# 09/26 ~ 09/30
+# 10/03 ~ 10/14
 
 - [Video Control]
   - Bug 1271765 - Visual refresh of media controls
-    - ask :astley for help with layout problem
-    - fix some mochitests and a11y tests, but still try to make them green on all platforms.
-    - a11y accessible tree is different from original XUL version. Get to tweak it in optimal way.
+    - review:
+      - Part 1: Layout reflow fix    r?dholbert
+      - Part 2: Visul refresh        r+jaws
+      - Part 3: Tests fix            r+jaws
+    - layout:
+      - discussed with :astley offline
+      - dholbert PTO til 10/17
+    - a11y:
+      - study ARIA and optimize new a11y tree
+      - get help from a11y team (:marcoZ and :surkov)
+      - ask them to verify new impl if it is still accessible
+    - l10n:
+      - don't use single quote in locale string(Bug 1294275) -> workaround with JavaScript
```

Comment:
yes. That bug is a similar intermittent bug shown on Treeherder, which making the same mistake as mine.
:jaws suggested me to use string like "#1 \<span class='duration'\>#2\</span\>" for locale friendly, but single quote could not be placed within locale string, we should use  ‘’ instead.